examples/tools/generate.ipynb

```diff
@@ -0,0 +1,502 @@
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "To illustrate this procedure, we first need to :doc:`create some kinematics\n",
+    "model <../workflow/1_create_model>`:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from pycompwa.expertsystem.ui.system_control import StateTransitionManager\n",
+    "from pycompwa.expertsystem.ui.system_control import InteractionTypes\n",
+    "from pycompwa.expertsystem.amplitude.helicitydecay import \\\n",
+    "    HelicityAmplitudeGeneratorXML\n",
+    "\n",
+    "initial_state = [(\"J/psi\", [-1, 1])]\n",
+    "final_state = [(\"gamma\"), (\"pi0\"), (\"pi0\")]\n",
+    "tbd_manager = StateTransitionManager(initial_state, final_state,\n",
+    "                                     formalism_type='helicity',\n",
+    "                                     topology_building='isobar')\n",
+    "\n",
+    "tbd_manager.set_allowed_interaction_types(\n",
+    "    [InteractionTypes.Strong, InteractionTypes.EM])\n",
+    "tbd_manager.allowed_intermediate_particles = ['f']\n",
+    "\n",
+    "graph_interaction_settings_groups = tbd_manager.prepare_graphs()\n",
+    "solutions, _ = tbd_manager.find_solutions(graph_interaction_settings_groups)\n",
+    "\n",
+    "print(\"found \" + str(len(solutions)) + \" solutions!\")\n",
+    "\n",
+    "xml_generator = HelicityAmplitudeGeneratorXML()\n",
+    "xml_generator.generate(solutions)\n",
+    "xml_generator.write_to_file('model.xml')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Now we can go through the procedure explained above:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pycompwa.ui as pwa"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# specify kinematics\n",
+    "particle_list = pwa.read_particles('model.xml')\n",
+    "kinematics = pwa.create_helicity_kinematics('model.xml', particle_list)\n",
+    "\n",
+    "# speficy generators\n",
+    "generator = pwa.EvtGenGenerator(\n",
+    "    kinematics.get_particle_state_transition_kinematics_info())\n",
+    "random_generator = pwa.StdUniformRealGenerator(12345)\n",
+    "\n",
+    "# generate phase space sample\n",
+    "phsp_sample = pwa.generate_phsp(1000, generator, random_generator)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Intensity-based samples"
+   ]
+  },
+  {
+   "cell_type": "raw",
+   "metadata": {
+    "raw_mimetype": "text/restructuredtext"
+   },
+   "source": [
+    "Data samples are more complicated than phase space samples in that they\n",
+    "represent the intensity profile resulting from a decay. You therefore need an\n",
```

Comment:
I would avoid saying decay, because it could also be some scattering process for example

Reply:
So rather stick with reaction